packages/e2e-test-utils/src/multi-select-blocks.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Select multiple blocks on the editor given the start clientId and the end clientId.
+ * The start block must be already selected.
+ *
+ * @param {string} start Identifier of the start block.
+ * @param {string} end Identifier of the end block.
+ */
+export async function multiSelectBlocks( start, end ) {
```

Comment:
Good idea 👍

Do you think it could be further abstracted, at the moment it’s only a proxy call to data layer and there is a helper for that already:


https://github.com/WordPress/gutenberg/blob/3156d0bbec4983bab009f7fe5dd1b5282df37d18/packages/e2e-test-utils/src/wp-data-select.js#L10

Reply:
There is still the need for having a `starting` reference. For example select from `selectMulti( 2, 5 )` that would select from the second block. I'll look into it! Nice idea @gziolo. I should also improve it by selecting the `start` block here.